source/release/4.5.0/index.md

```diff
@@ -72,69 +76,72 @@ dnf config-manager --set-enabled ovirt-45-upstream-testing
+To check if provider and OVN was configured successfully on the host, webadmin includes OVN configured flag in General tab for every host, at the same time this value is available via REST API. Please note that this value might say "No" until capabilities are refreshed for the host.
 
-Fixing the host's OVN not being confiugred can be done by reinstalling it from engine 4.5 or higher.
- - [BZ 1782056](https://bugzilla.redhat.com/show_bug.cgi?id=1782056) **[RFE] Integration of built-in ipsec feature in RHV/RHHI-V with OVN**
+Fixing the host's OVN not being configured can be done by reinstalling it from engine 4.5 or higher.
+ - [BZ 1782056](https://bugzilla.redhat.com/show_bug.cgi?id=1782056) **[RFE] Integration of built-in ipsec feature in oVirt with OVN**
    The IPSec for OVN feature is available on hosts with configured ovirt-provider-ovn, OVN version 2021 or higher and OvS version 2.15 or higher.
 
 #### oVirt Open vSwitch
 
- - [BZ 1782056](https://bugzilla.redhat.com/show_bug.cgi?id=1782056) **[RFE] Integration of built-in ipsec feature in RHV/RHHI-V with OVN**
+ - [BZ 1782056](https://bugzilla.redhat.com/show_bug.cgi?id=1782056) **[RFE] Integration of built-in ipsec feature in oVirt with OVN**
    The IPSec for OVN feature is available on hosts with configured ovirt-provider-ovn, OVN version 2021 or higher and OvS version 2.15 or higher.
- - [BZ 1940824](https://bugzilla.redhat.com/show_bug.cgi?id=1940824) **[RFE] Upgrade OVN/OVS 2.11 in RHV to OVN/OVS 2.15**
+ - [BZ 1940824](https://bugzilla.redhat.com/show_bug.cgi?id=1940824) **[RFE] Upgrade OVN/OVS 2.11 in oVirt to OVN/OVS 2.15**
    Upgrade from OvS/OVN 2.11 to OVN 2021 and OvS 2.15.
 
 The upgrade is transparent to user as long as few conditions are met:
 
 1) The engine must be upgraded first.
 2) Before any host upgrade the ovirt-provider-ovn security groups must be disabled for all OVN networks that are expected to work between host with OVN/OvS version 2.11.
 3) Upgrade hosts to match the OVN version 2021 or higher and OvS version to 2.15. This step should be done via the UI, in order to reconfigure OVN and refresh certificates.
+4) Reboot the host after upgrade
 
 
-To check if provider and OVN was configured successfully on the host, webadmin includes OVN configured flag in General tab for every host, at the same time this value is available via REST API. 
+To check if provider and OVN was configured successfully on the host, webadmin includes OVN configured flag in General tab for every host, at the same time this value is available via REST API. Please note that this value might say "No" until capabilities are refreshed for the host.
 
-Fixing the host's OVN not being confiugred can be done by reinstalling it from engine 4.5 or higher.
+Fixing the host's OVN not being configured can be done by reinstalling it from engine 4.5 or higher.
```

Comment:
Suggestion: "If the host's OVN is not configured, you can fix this issue by reinstalling the host with engine version 4.5 or later."